test/e2e/util/e2erequest.go

```diff
@@ -17,7 +19,20 @@ const (
 )
 
 func buildURL(apiPath string) string {
-	return fmt.Sprintf("http://localhost:%d%s", APIPort, apiPath)
+	env_sippy_api_port := os.Getenv("SIPPY_API_PORT")
+	env_sippy_endpoint := os.Getenv("SIPPY_ENDPOINT")
```

Comment:
nit, go style is mixed caps

https://go.dev/doc/effective_go#mixed-caps

Reply:
LOL, too much bash lately -- will fix.